### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: README.md [hunk 1]
```diff
@@ -39,16 +34,6 @@ Python and JavaScript. It aims to be a drop-in replacement for
 
 ![Screenshot of Isso](./doc/screenshots/example.jpg)
 
-## Disclaimers / important information
-
-* Limitations
-    * Requires a dedicated domain
-
-* Extra information
-    * The admin interface for the app is `https://yourdomain.tld/admin`
```
Comment (navanchauhan) [inline README.md:49]: Are we documenting this somewhere else?

File: conf/isso.cfg [hunk 2]
```diff
@@ -16,12 +16,12 @@ password =
...
 security = none
-to = __ADMINMAIL__
+to = __EMAIL__
 from = "Isso Comments" <isso@__DOMAIN__>
 timeout = 10
 
 [server]
-listen = http://localhost:__PORT__
+listen = http://127.0.0.1:__PORT__
 public-endpoint =
```
Comment (navanchauhan) [inline conf/isso.cfg:27]: I think it will be better to add the trusted-proxies option here, what do you think? Might be redundant

**Review by navanchauhan — COMMENTED** [review]
Otherwise looks good to me, will need to manually test it though

File: README.md (hunk 1 above)
Comment (alexAubin) [reply README.md:49]: (marking as resolved, cf doc/ADMIN.md)

**Review by alexAubin — APPROVED** [review]

**Comment by alexAubin** [comment]
!testme

**Pull request merged**